Sources/CoreMetrics/Metrics.swift

```diff
@@ -160,6 +136,10 @@ public class Recorder {
 
     /// Record a value.
     ///
+    /// Recording a value is meant to have "set" semantics, rather than "add" semantics.
```

Comment:
```suggestion
    /// Recording a value is meant to have "set" semantics, rather than "add" semantics.
    /// This means that the value of this `Recorder` will match whatever you pass in here.
```

Took a swing at clarifying `"set" semantics` which I understand, but not sure that's immediately clear without some extra context to all folks.

Reply:
I personally think just defining `"set"` semantics is clear enough.

Reply:
this looks good to me